src/components/ui/Button.astro

```diff
@@ -1,9 +1,30 @@
 ---
-import SHA from './SHA.astro';
+import { cva, type VariantProps } from 'class-variance-authority';
 
-interface Props {
-  color?: 'orange' | 'green' | 'lightblue' | 'semidarkblue';
-  size?: 'sm' | 'lg' | 'lg-full' | 'sm-mobfull';
+import SHA from '@/components/ui/SHA.astro';
+import Link from '@/components/util/Link.astro';
+
+export const buttonVariants = cva(
+  'font-bold text-white drop-shadow transition-colors rounded-full text-center',
+  {
+    variants: {
+      color: {
+        orange: 'bg-nix-orange hover:bg-nix-orange-hover',
+        lightblue: 'bg-nix-blue-light hover:bg-nix-blue-light-hover',
+        semidarkblue: 'bg-nix-blue-dark hover:bg-nix-blue-dark-hover',
+        green: 'bg-nix-green hover:bg-nix-green-hover',
```

Comment:
In my testing I couldn't really tell how the hover state applied, am I missing something or is it just to subtile? I'd guess the mentioned colors may need to be defined first. :D